docs/content.zh/docs/connectors/table/hbase.md

```diff
@@ -38,7 +38,7 @@ HBase 连接器在 upsert 模式下运行，可以使用 DDL 中定义的主键
 依赖
 ------------
 
-{{< sql_download_table "hbase" >}}
```

Comment:
I think this is unnecessary. 
Because HBase connector isn't yet externalized from Flink itself. 

You can ref the comments of this jira https://issues.apache.org/jira/browse/FLINK-31671.

Reply:
OK, I think that also makes sense.